src/main/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheck.java

```diff
@@ -83,6 +83,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <p>
  * In default configuration the check does not rise any violations.
```

Comment:
`To configure the default check that does not ....`

Reply:
Done